### PR TITLE
Add GitHub Actions for automated deployments

### PR DESCRIPTION
**Pull request by 2ndkauboy**
To be merged before the next release.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,10 +1,15 @@
-* text eol=lf
-*.jpg -text
-*.png -text
-*.gif -text
```
Comment (bueltge) [inline .gitattributes]: I mean we should leave them because we get several problems with media files inside the repository if we remove them.

**Review by bueltge — COMMENTED** [review]
@2ndkauboy small question or note, maybe you can validate them and after that we should merge them. @krafit

File: .gitattributes [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
-.stylelintrc.json            export-ignore
+# Files
+/.eslintrc.json export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.stylelintrc.json export-ignore
+/.travis.yml export-ignore
+/composer.json export-ignore
+/composer.lock export-ignore
+/phpcs.xml export-ignore
```
Comment (patrickrobrecht) [inline .gitattributes:19]: I think that the files `CHANGELOG.md`, `package.json`, `package-lock.json` and `README.md` (we have a separate `readme.txt` for wp.org) and the directories `node_modules` and `vendor` should be added as `export-ignore`, right?

**Review by patrickrobrecht — CHANGES_REQUESTED** [review]

File: .gitattributes (hunk 2 above)
Comment (stklcode) [reply .gitattributes:19]: `node_modules` and `vendor` are already listed above under _directories_

Changelog is at least not completely useless, but not really necessary either.

**Review by patrickrobrecht — APPROVED** [review]

File: .gitattributes (hunk 2 above)
Comment (patrickrobrecht) [reply .gitattributes:19]: Added the missing files to the `.gitattributes` file.

**Review by patrickrobrecht — APPROVED** [review]

**Pull request merged**